### PR TITLE
Hit api

### PR DESCRIPTION
**Pull request by basnetlaxmi**
In this milestone, I :

-Used Async and Await
- Fetch data from API

### CODE REVIEW

File: src/apidata.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/Zkfjdkfhuefhfhf/scores/';
+
+async function postData(data) {
+  const response = await fetch(url, {
```
Comment (bimbolabuari) [inline src/apidata.js:4]: Let's use an arrow function instead of the `function` keyword ✊  for best practice.

File: src/apidata.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    method: 'POST',
+    mode: 'cors',
+    headers: { 'Content-type': 'application/json' },
+    body: JSON.stringify(data),
+  });
+  return response.json();
+}
+
+async function getData() {
```
Comment (bimbolabuari) [inline src/apidata.js]: Let's use an arrow function instead of the `function` keyword ✊  for best practice

File: src/data.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import { getData, postData } from './apidata.js';
+
+// eslint-disable-next-line import/no-mutable-exports
```
Comment (bimbolabuari) [inline src/data.js]: Kindly remove disabled linters for a cleaner and professional code 🙂

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @basnetlaxmi , 

You have done a good job so far 🚀  

## Highlights 🎉 

- Use async and await with the fetch method to send and receive data.

There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

File: README.md [hunk 4]
```diff
@@ -1 +1,56 @@
...
+- GitFlow 
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+1) Enter `git@github.com:basnetlaxmi/leaderboard.git` into terminal
+
+2) Open in browser to view site
```
Comment (julie-ify) [inline README.md:22]: - [OPTIONAL]
Consider adding instructions on how to run your app when someone clones it to their local machine, something like run `npm install` to install all the dependencies and `npm start ` to run the app in the browser.

File: src/data.js [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+function display(data) {
+  const table = document.querySelector('.scores-box');
+  table.innerHTML = '';
+  const codeData = data.map((el, i) => `<li id= "${index(i)}" class= "table-data">
+  <span class="name">${el.user}</span>
+  <span class="score">${el.score}</span>
+  </li>`).join('');
+  table.innerHTML = codeData;
+}
```
Comment (julie-ify) [inline src/data.js:20]: - [OPTIONAL] 

Try changing this function keyword to an arrow function also, It was not highlighted by the previous reviewer and that is why I marked it as optional but please try to make the changes 😃

**Review by julie-ify — APPROVED** [review]
## Hi @basnetlaxmi,

# Status: Approved ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
Approved emoji




Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**